src/bankAccount.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, amount, birthday) {
```

Comment:
you don't need birthday here

Reply:
- The person has the following initial info: name, date of birth in string format, initial amount of money. - from readme/ deleted

Reply:
sorry, it isn't used that's why I forgot about it